lib/mix/test/mix/tasks/compile_test.exs

```diff
@@ -157,4 +169,20 @@ defmodule Mix.Tasks.CompileTest do
   after
     Application.delete_env(:erl_config_app, :value)
   end
+
+  test "compile a project with wrong path" do
```

Comment:
Tests should be in present tense, not in imperative. So it should read:
"compiles a project"

Reply:
English is not my first language but if you open the file lib/mix/test/mix/tasks/compile_test.exs there're a lot tests with "compile" they are wrong too?

Reply:
Yes, they should be corrected.
I think it was myself and/or @lexmag who standardized all the tenses but those that you see have been introduced relatively recently.
You can see it has been done before in c6f83b5 for example